src/bin/check-icu4c-version.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+if [ $# -ne 0 ];
+then
+    cd $1
+    shift
+fi
+
+if [ ! -f  ./source/common/unicode/uversion.h ];
```

Comment:
The initial `./` doesn't add any value here, the current directory is always the base for any relative path.